### PR TITLE
Fix typo in FindMOAB.cmake

### PR DESCRIPTION
**Pull request by bam241**
small typo preventing correct inclusion of `MOAB_INCLUDE_DIRS` in my `CMAKE_INCLUDE_DIRS`

### CODE REVIEW

**Comment by zxkjack123** [comment]
Do we have a docker image for MacOS CircleCI?

**Comment by bam241** [comment]
Sadly not... I just fix it to work on my cokputer

BaM 
from my phone

File: cmake/FindMOAB.cmake [hunk 1]
```diff
@@ -49,7 +49,7 @@ else (MOAB_LIBRARIES AND MOAB_INCLUDE_DIRS)
   # handle the QUIETLY and REQUIRED arguments and set MOAB_FOUND to TRUE
   # if all listed variables are TRUE
   find_package_handle_standard_args(MOAB  DEFAULT_MSG
-                                    MOAB_LIBRARY MOAB_INCLUDE_DIR)
+      MOAB_LIBRARY MOAB_INCLUDE_DIRS)
```
Comment (gonuke) [inline cmake/FindMOAB.cmake:52]: This looks suspicious- should it be `MOAB_LIBRARIES` as well?

**Review by gonuke — COMMENTED** [review]
Too many variables with similar names...

**Comment by gonuke** [comment]
@bam241 - can you confirm whether a change like this is still necessary after all your testing on macOS?

**Comment by ahnaf-tahmid-chowdhury** [comment]
@bam241, are you still working on this? I think we have made a lot of updates on this. Make a rebase?

**Comment by bam241** [comment]
Sadly I don't have time to this with this.

File: cmake/FindMOAB.cmake (hunk 1 above)
Comment (ahnaf-tahmid-chowdhury) [reply cmake/FindMOAB.cmake:52]: We have already changed to MOAB_INCLUDE_DIRS through PR #1104

Comment (ahnaf-tahmid-chowdhury) [reply cmake/FindMOAB.cmake:52]: @bam241, You can proceed to close this if you want. As it is already changed in latest PyNE.

**Pull request closed without merging**